paper-spinner-styles.js

```diff
@@ -238,6 +238,19 @@ $_documentContainer.innerHTML = `<dom-module id="paper-spinner-styles">
         overflow: hidden;
       }
 
+      .circle-clipper .circle {
```

Comment:
In my branch, I combined most of these properties with `.spinner-layer::after` (like it was before).

I also had to add an empty `export {}` statement for some internal compatibility.

Reply:
I see. Thank you for getting this to a finish line